scripts/analytics/mongodb/averageFirstWeekPlaytime.js

```diff
@@ -0,0 +1,76 @@
+// Usage: mongo <level-session-db-host> -u <user> -p <pwd> averageFirstWeekPlaytime.js
+
+var sTime = (new Date()).getTime();
+
+var auth = JSON.parse(cat('.mainAuth.json'));
+var main = connect(auth.main.uri, auth.main.user, auth.main.password);
+
+// Find all users who were created between 2 weeks ago and last week
+
+const START_DAYS_AGO = 14;
+const END_DAYS_AGO = 7;
+const WINDOW_DAYS = 7;
+
+function objectIdFromDate(date) {
+  return ObjectId(Math.floor(date.getTime() / 1000).toString(16) + "0000000000000000");
+}
+
+print("Getting all users created between", START_DAYS_AGO, "and", END_DAYS_AGO, "days ago...");
+
+var twoWeeksAgo = new Date();
+twoWeeksAgo.setDate(twoWeeksAgo.getDate() - START_DAYS_AGO);
+twoWeeksAgo.setHours(0, 0, 0, 0);
```

Comment:
I'd recommend having a 'start date' which can either be hard coded or handed in via the command line. That way the script doesn't *have* to be run on the specific day. [For example](https://github.com/codecombat/codecombat/blob/master/scripts/analytics/projectShares.js#L32).